core/sdk/src/api/context.rs

```diff
@@ -141,14 +141,25 @@ impl RippleContext {
                 false
                 // This is not an update request so need not to honour it
             }
-            RippleContextUpdateRequest::Features(features) => {
+            RippleContextUpdateRequest::UpdateFeatures(features) => {
```

Comment:
Improved this a bit so we can pass in one or more features and indicate whether or not they're en/disabled.